samples/mixed/consumer/src/main.rs

```diff
@@ -15,24 +15,49 @@ use samples_protobuf_data_access::sample_grpc::v1::digital_twin_provider::digita
 use samples_protobuf_data_access::sample_grpc::v1::digital_twin_provider::{
     InvokeRequest, SetRequest, SubscribeRequest,
 };
+use serde_derive::{Deserialize, Serialize};
 use std::net::SocketAddr;
 use tokio::time::{sleep, Duration};
 use tonic::{Status, transport::Server};
 use uuid::Uuid;
 
+#[derive(Debug, Serialize, Deserialize)]
+struct IsAirConditioningActiveProperty {
+    #[serde(rename = "IsAirConditioningActive")]
+    is_air_conditioning_active: sdv::hvac::is_air_conditioning_active::TYPE,
+    #[serde(rename = "$metadata")]
+    metadata: Metadata,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+struct ShowNotificationRequestPayload {
+    #[serde(rename = "Notification")]
+    notification: sdv::hmi::show_notification::request::TYPE,
+    #[serde(rename = "$metadata")]
+    metadata: Metadata,
+}
+
 /// Start the show-notification repeater.
 ///
 /// # Arguments
 /// `provider_uri` - The provider_uri.
 /// `consumer_uri` - The consumer_uri.
 fn start_show_notification_repeater(provider_uri: String, consumer_uri: String) {
     debug!("Starting the Consumer's show-notification repeater.");
+
     tokio::spawn(async move {
-        loop {
-            let payload: String = "show-notification request".to_string();
+        let metadata = Metadata { model: sdv::hmi::show_notification::request::ID.to_string() };
+
+        let request_payload: ShowNotificationRequestPayload = ShowNotificationRequestPayload {
```

Comment:
the type annotation here is not necessary

Reply:
I prefer to make it explicit.  Leaving it as is.